examples/datasets/plot_limo_data.py

```diff
@@ -111,68 +169,250 @@
 # We can also compute the difference wave contrasting Face A and Face B.
 # Although, looking at the evoked responses above, we shouldn't expect great
 # differences among these face-stimuli.
-#
-# Compute difference wave (Face A minus Face B)
-difference_wave = mne.combine_evoked([limo_epochs['Face/A'].average(),
-                                     -limo_epochs['Face/B'].average()],
-                                     weights='equal')
 
-# Plot difference between Face A and Face B
+# Face A minus Face B
+difference_wave = combine_evoked([limo_epochs['Face/A'].average(),
+                                  -limo_epochs['Face/B'].average()],
+                                 weights='equal')
```

Comment:
Sorry for bringing this up again, but wasn't the plan to switch to using weights for subtracting evokeds?

@larsoner

Reply:
soo, more like this:

`combine_evoked([limo_epochs['Face/A'].average(), limo_epochs['Face/B'].average()], weights=[1, -1]`

Reply:
no.  As the docstring says:

> Subtraction can be performed by calling `combine_evoked([evoked1, -evoked2], 'equal')`

Using `weights` is considered "expert mode" and is the reason for the warning (also in the docstring):

> Warning: If you provide an array of weights instead of using ‘equal’ or ‘nave’, strange things may happen with your resulting signal amplitude and/or *nave* attribute.

Reply:
alright, so IIUC we keep it as it was ...?

Reply:
yes. keep as it is currently in this PR.